test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -960,6 +962,9 @@ describe('ReactRefreshLogBox app', () => {
            |           ^",
          "stack": [
            "{default export} index.js (3:11)",
+           "Set.forEach <anonymous> (0:0)",
+           "<FIXME-file-protocol>",
+           "<FIXME-file-protocol>",
```

Comment:
These issues where masked due to manually cutting off at `react-stack-bottom-frame`. Might reintroduce that later in the error overlay or middleware. But it's really a bug with sourcemaps in Turbopack that needs fixing so that